src/site/content/en/blog/badging-api/index.md

```diff
@@ -86,67 +85,63 @@ Examples of sites that may use this API include:
 4. Click the **Set** or **Clear** button to set or clear the badge from the app
    icon. You can also provide a number for the *Badge value*.
 
-{% Aside %}
-  While the Badging API *in Chrome* requires an installed app
-  with an icon that can actually be badged, you shouldn't
-  make calls to the Badging API dependent on the install state.
-  The Badging API can apply to *anywhere* a browser might want to show a badge,
-  so developers shouldn't make any assumptions about situations where
-  the browser will display them. Just call the API when it exists.
-  If it works, it works. If not, it simply doesn't.
-{% endAside %}
-
 ## How to use the Badging API {: #use }
 
-Starting in Chrome 73, the Badging API is available as an origin trial
-for Windows (7+) and macOS.
+The Badging API is available as an origin trial in Chrome 79 and later.
 
-{% Aside 'caution' %}
-  Android is not supported because it requires you to show a notification,
-  though this may change in the future. Chrome OS support is pending
-  implementation of badging on the platform.
-{% endAside %}
+The Badging API works on Windows, and macOS. Support for ChromeOS is in
+development and will be available in a future release of Chrome.
+Unfortunately, Android is not supported because it requires you to show a
+notification, though this may change in the future.
 
 ### Register for the origin trial {: #ot }
 
+{% Aside 'caution' %}
+In Chrome 79 (stable Dec 2019), the first origin trial ended, and was replaced
+by a new origin trial that uses a slightly different API design. Unfortunately,
+it is not backwards compatible, and calls to the previous API will fail. See
+[Supporting both the first and second origin trial](#backwards-compat) for
+details on how to provide backwards compatibility.
+{% endAside %}
+
 {% include 'content/origin-trials.njk' %}
 
 {% include 'content/origin-trial-register.njk' %}
 
+If necessary, you can
+[request a token](https://developers.chrome.com/origintrials/#/view_trial/1711367858400788481)
```

Comment:
I think we're planning on removing this when M79 goes stable (but I'm not 100% sure).

Reply:
According to @mgiuca, sounds like it's staying.

Reply:
:+1: